src/components/Sidebar.tsx

```diff
@@ -12,18 +12,29 @@ import MoreHorizon from "@material-ui/icons/MoreHoriz";
 import "./Sidebar.css";
 import { Button } from "@material-ui/core";
 
-const Sidebar = () => {
+interface SidebarOption {
+  Icon: React.ElementType;
+  text: string;
+}
+
+const Sidebar: React.FC = () => {
+  const sidebarOptions: SidebarOption[] = [
+    { Icon: HomeIcon, text: "Home" },
+    { Icon: SearchIcon, text: "Search" },
+    { Icon: NotificationsNoneIcon, text: "Notifications" },
+    { Icon: MailOutlineIcon, text: "Mail" },
+    { Icon: BookmarkBorderIcon, text: "Bookmarks" },
+    { Icon: ListAltIcon, text: "Lists" },
+    { Icon: PermIdentityIcon, text: "Identity" },
+    { Icon: MoreHorizon, text: "More" },
```

Comment:
Did you not need to import these icons?

Reply:
imported and stored in a list to map over each object. Please let me know how can I improve it @roopaljasnani

Reply:
NVM. I saw the entire file contents just now, saw the imports also. 👍🏻